### DISCUSSION TITLE
Getting Started in Verde?

### DISCUSSION THREAD

**Discussion by MetGrad** [fatiando/community#74] [Q&A] [answered]
I'm a meteorology student and I need to use Verde to combine two datasets of ocean surface currents. Each dataset has a different grid and grid spacing. I'm new to Verde and any kind of interpolations. I've been reading the list of functions and going through the examples. Any tips or suggestions for getting started on a problem like this?

**Comment by santisoler** [comment, marked as answer]
Hi @MetGrad ! Thanks for opening this discussion. You can always use Verde for interpolating grids, there's no technical problem behind it. But since you have already gridded data (and specially if you have them as xarray objects), you might want to use [xarray interpolation methods](https://docs.xarray.dev/en/stable/user-guide/interpolation.html#multi-dimensional-interpolation)

Check [this other discussion topic](https://github.com/orgs/fatiando/discussions/72#discussioncomment-3990766) for more information.
